ch03.asciidoc

```diff
@@ -209,9 +209,11 @@ As new addition, a channel partner will encumber their own output with a timeloc
 The timelock prevents the owner of the output to spend it immediately once the commitment transaction was included in a block.
 The timelock is usually measured in blocktime and can be up to 2016.
 On average, 2016 blocks represent or last two weeks (assuming a blocktime of 10 minutes as in the Bitcoin network).
+Be aware, the exact number of blocks used for the timeout can be set to different values by different software.
+2016 is just the maximum value, but smaller values are possible.
```

Comment:
Can mention here that one likely wants a longer value the larger the channel size. 2016 is just what's used as the canonical example.

Reply:
suggestion incorporated